src/lua/api-gateway/redis/redisConnectionProvider.lua

```diff
@@ -61,6 +62,11 @@ end
 -- Redis authentication
 function RedisConnectionProvider:getConnection(upstream)
     local redis = restyRedis:new()
+
+    -- Introduce monitoring
```

Comment:
Can we set this behind a flag ? 
IIRC os.time has performance impacts, and so does extra logging.

Reply:
See this conversation as well: https://github.com/openresty/lua-nginx-module/issues/598